### PR TITLE
chore(wren-ai-service): add correlation ID to langfuse

### PR DESCRIPTION
**Pull request by grieve54706**
Add correlation ID from the ibis server to the Langfuse.

The AI team usually provides the issues from the Langfuse, if we want to find the ibis log for the issue, we need to find it by time, and the time is not completely correct.

The correlation ID is from the ibis-server. The AI service gets data and the correlation ID through the web API, and the AI service sends Langfuse logs. So, we must explain the correlation ID and return it to the AI service.

### CODE REVIEW

**Comment by grieve54706** [comment]
Hi @onlyjackfrost, @cyyeh. Could you check this PR?

File: wren-ui/src/apollo/server/services/queryService.ts [hunk 1]
```diff
@@ -28,8 +28,8 @@ export interface ColumnMetadata {
 export interface PreviewDataResponse {
   correlationId?: string;
   processTime?: string;
-  columns: ColumnMetadata[];
-  data: any[][];
+  columns?: ColumnMetadata[];
```
Comment (onlyjackfrost) [inline wren-ui/src/apollo/server/services/queryService.ts:31]: I prefer not to share type definitions here, as preview and dry-run have distinct return formats, and they should be defined separately. Can we define their types individually?

**Review by onlyjackfrost — COMMENTED** [review]
Could you please explain why the correlationId is being added and returned to the AI service?

File: wren-ui/src/apollo/server/services/queryService.ts (hunk 1 above)
Comment (grieve54706) [reply wren-ui/src/apollo/server/services/queryService.ts:31]: Sure

**Comment by grieve54706** [comment]
> Could you please explain why the correlationId is being added and returned to the AI service?

Hi @onlyjackfrost. Because the AI team usually provides the issues from the Langfuse, if we want to find the ibis log for the issue, we need to find it by time, and the time is not completely correct. 
The correlation ID is from the ibis-server. The AI service gets data and the correlation ID through the web API, and the AI service sends Langfuse logs. So, you must explain the correlation ID and return it to the AI service.

**Review by onlyjackfrost — APPROVED** [review]

**Pull request merged**